### ISSUE TITLE
Preproc steps

### ISSUE THREAD

**Issue by ahmed-moubtahij** [tahmedge/QR-BERTSUM-TL-for-QFAS#3]
Are you using cross-validation to fine-tune QR-BERTSUM-TL or should I merge all preprocessed DebatePedia train, valid and test files into one folder as per the original BertSum?

**Comment by tahmedge** [comment]
No, it should not be merged together. For each fold of the dataset, the model was trained separately. Then, based on the performance in the validation set, the best model for each fold was selected for prediction in the test data. The final result is the average across 10 folds

**Comment by ahmed-moubtahij** [comment]
Understood, thanks for the prompt answer!
